CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift

```diff
@@ -19,7 +19,8 @@ public let ATTRIBUTE_NODES: [Node] = [
     element: "Syntax",
     elementName: "Attribute",
     elementChoices: ["Attribute", "IfConfigDecl"],
-    omitWhenEmpty: true
+    omitWhenEmpty: true,
+    elementsSeparatedByNewline: true
```

Comment:
We may need `elementsSeparatedBySpace` instead… Splitting the attribute list with newlines looks terrible, especially when they’re applied to a declaration without leading newline trivia.